performance/config.py

```diff
@@ -16,6 +16,9 @@
 class Config:
     ENV = 'prod'
 
+    REPORTS_S3_BUCKET = 'govukverify-hub-prod-billing-reports'
+    BILLING_DIRECTORY = 'billing'
+    VERIFICATIONS_DIRECTORY = 'rp'
```

Comment:
The name of the var needs to be clear that we are talking about the remote directory on S3. Probably an `S3_` prefix would do.